download.sh

```diff
@@ -0,0 +1,174 @@
+#! /usr/bin/env bash
+USAGE="Usage: ./download.sh [-hD] [-c <NUM>] <DUMP_DIR>
+
+Download the latest Wikipedia Enterprise HTML dumps.
+
+Arguments:
+    <DUMP_DIR>  An existing directory to store dumps in. Dumps will be grouped
+                into subdirectories by date, and a link 'latest' will point to
```

Comment:
Will wikiparser generator properly find/load newer versions from the latest dir without specifying explicit file names?

Reply:
For the `run.sh` script, you'll provide a glob of the latest directory:
```
./run.sh descriptions/ planet.osm.pdf $DUMP_DIR/latest/*
```
It doesn't have any special handling for the `$DUMP_DIR` layout.